source/install-from-source.txt

```diff
@@ -0,0 +1,373 @@
+clone the repository using the command line, the following command may be used:
+
+.. code-block:: bash
+
+  $ git clone https://github.com/mongodb/mongo-c-driver.git --branch="$VERSION" "$SOURCE"
+
+.. tip::
+
+  Despite the name, ``git-clone``'s ``--branch`` argument may also be used to
+  clone from repository *tags*.
+
+.. _learn.obtaining.archive:
+
+Downloading a Release Archive
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An archived snapshot of the repository can be obtained from the
+`GitHub Releases Page <https://github.com/mongodb/mongo-c-driver/releases>`__.
+The ``mongo-c-driver-x.y.z.tar.gz`` archive attached to any release contains the
+minimal set of files that you'll need for the build.
+
+.. tabs::
+
+   .. tab:: Using ``wget`` + ``tar``
+      :tabid: wget-tar
+
+      .. code-block:: none
+        
+        ## Download using wget:
+        $ wget "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.tar.gz" \
+             --output-document="mongo-c-driver-$VERSION.tar.gz"
+        ## Extract using tar:
+        $ tar xf "mongo-c-driver-$VERSION.tar.gz"
+
+   .. tab:: Using ``curl`` + ``tar``
+      :tabid: curl-tar
+
+      .. code-block:: none
+        
+        ## Using curl:
+        $ curl "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.tar.gz" \
+             --output="mongo-c-driver-$VERSION.tar.gz"
+        ## Extract using tar:
+        $ tar xf "mongo-c-driver-$VERSION.tar.gz"
+
+   .. tab:: PowerShell
+      :tabid: powershell
+
+      .. code-block:: none
+
+        ## Use Invoke-WebRequest:
+        PS> $url = "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.zip"
+        PS> $file = "mongo-c-driver-$VERSION.zip"
+        PS> Invoke-WebRequest -UseBasicParsing -Uri $url -OutFile $file
+        ## Extract using Expand-Archive:
+        PS> Expand-Archive mongo-c-driver-$VERSION.zip
+
+The above commands will create a new directory ``mongo-c-driver-$VERSION`` within
+the directory in which you ran the ``tar`` / ``Expand-Archive`` command (**note**:
+PowerShell will create an additional intermediate subdirectory of the same
+name). This directory is the root of the driver source tree (which we refer to
+as ``$SOURCE`` in these documents). The ``$SOURCE`` directory should contain the
+top-level ``CMakeLists.txt`` file.
+
+
+Obtaining Prerequisites
+-----------------------
+
+In order to build the project, a few prerequisites need to be available.
+
+Both ``libmongoc`` and ``libbson`` projects use `CMake <https://cmake.org>`__ for build configuration.
+
+.. note::
+
+   It is *highly recommended* -- but not *required* -- that you download the
+   latest stable CMake available for your platform.
+
+.. tip:: Getting the Latest CMake
+   
+   A new stable release of CMake can be obtained from
+   `the CMake downloads page <https://cmake.org/download/#latest>`__.
+
+   For Windows and macOS, simply download the CMake ``.msi`` / ``.dmg`` (not the
```

Comment:
I: [slashes](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#slashes)

```suggestion
   For Windows and macOS, simply download the CMake ``.msi`` or ``.dmg`` (not the
```

(using tabs would also solve this issue)